### ISSUE TITLE
Play space issue

### ISSUE THREAD

**Issue by CinderVenus** [alvr-org/alvr-visionos#108]
I seem to be having issues where my play space is always off centered and a few feet below me. I have tried using Continuous Calibration to try and re-center myself. I have tried to not use that and use regual space calibraion. I tried messing with setting in the alvr launcher from position recentering mode being disabled, local floor and local, as well the rotation recentering mode disabled, yaw and tilted. I have tried uninstalling my steam vr, resetting my play area, going into developer settings and quick calibrating, I have tried running ovr advanced setting to drag myself and that is not fruitful as well. I am sunk in the floor in vrchat and about 8 feet and to like 8 inches to my right above my steam vr overlay when I get in.

**Comment by shinyquagsire23** [comment]
i vaguely recall the standing space being kinda weird, had to launch Virtual Desktop or set up the bounds with lighthouse trackers w/o ALVR? Not 100% on that though. I think ALVR has some weird settings wrt floors maybe.

If you want it to keep the SteamVR center you'll need to keep "SteamVR ignores crown button long press" set to enabled. You can also try and reset the SteamVR centering anchor by long pressing 3 times in a row (might also be worth uninstalling the visionOS app and reinstalling)

I've also had the floor sink a bunch when I accidentally had a glass PC case in the same room as the lighthouse basestations.

**Comment by CinderVenus** [comment]
yeah it is odd that neither my index or quest pro have these issues. Especially my Quest pro setup as I use virtual desktop and my continuous calibration to be in vrchat normally, and never have these issues. It seem even when I was messing with ovr to tweak my standing position by rotating and moving the floor up and down, the relative position of my tundra tracker and base stations would never move but my steam vr space would, but not in vrchat. I did try re installing the client on the vision pro and long pressing the crown as you said multiple times.

**Comment by CinderVenus** [comment]
Were you talking about redoing my room setup with the vision pro, by having a tundra tracker on the ground when it calls for the hmd to be on the ground?

**Comment by shinyquagsire23** [comment]
Hmm yeah no clue, I ~vaguely recall OVR advanced settings causing issues maybe? Because it kept loading spaces in which conflicted or something. Probably best to keep extensions and such to a minimum.

**Comment by tappdarden** [comment]
Did you get it figured out? i'm having the same issue where i'm floating way to high in the air. I tried all these things and nothing seems to move me from being stuck in the ceiling.

**Comment by ifokeev** [comment]
Same issue here; I haven't found a fix yet :(
